numpy/core/_add_newdocs.py

```diff
@@ -829,6 +830,16 @@
         Specifies the minimum number of dimensions that the resulting
         array should have.  Ones will be pre-pended to the shape as
         needed to meet this requirement.
+    like : array_like
+        Reference array-like object created by a downstream library, such
+        as CuPy, Dask, xarray, etc. If specified, the function will be
+        dispatched to the downstream library, provided it supports the
+        __array_function__ protocol. This will bypass the NumPy's
```

Comment:
```suggestion
        ``__array_function__`` protocol. This will bypass the NumPy's
```

Reply:
This is not the final wording, so I'm deferring this change for now and will apply it once we agree on the wording. Latest discussion in https://github.com/numpy/numpy/pull/16935#discussion_r466030901 .